cleverhans/utils_tf.py

```diff
@@ -198,7 +198,6 @@ def model_eval(sess, x, y, predictions=None, X_test=None, Y_test=None,
              the learning phase of a Keras model for instance.
     :param args: dict or argparse `Namespace` object.
                  Should contain `batch_size`
-    :param model: (deprecated) if not None, holds model output predictions
```

Comment:
The model argument is still present (in the `model_eval` function definition above)

Reply:
Good catch. Fixed